tests/unit/require.ts

```diff
@@ -684,6 +684,21 @@ registerSuite({
 				assert.deepEqual(pluginConfig.paths, paths, 'Paths should be equal');
 				assert.deepEqual(pluginConfig.map, map, 'Map should be equal');
 			}));
+		},
+
+		relativePluginPaths() {
+			let dfd = this.async(DEFAULT_TIMEOUT);
```

Comment:
can change, no worries, all the others are using let, so this was copy/pasted, happy to update all